src/dmd/backend/cv8.d

```diff
@@ -5,65 +5,82 @@
  * Copyright:    Copyright (C) 2012-2018 by The D Language Foundation, All Rights Reserved
  * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
  * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
- * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/code.c, backend/code.c)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/cv8.d, backend/cv8.d)
  */
 
 // This module generates the .debug$S and .debug$T sections for Win64,
 // which are the MS-Coff symbolic debug info and type debug info sections.
 
-#if !SPP
-
-#include        <stdio.h>
-#include        <string.h>
-#include        <stdlib.h>
-#include        <time.h>
-
-#include        "cc.h"
-#include        "el.h"
-#include        "code.h"
-#include        "oper.h"
-#include        "global.h"
-#include        "type.h"
-#include        "dt.h"
-#include        "exh.h"
-#include        "cgcv.h"
-#include        "cv4.h"
-#include        "obj.h"
-#include        "outbuf.h"
-#include        "varstats.h"
-
-static char __file__[] = __FILE__;      /* for tassert.h                */
-#include        "tassert.h"
-
-#if _MSC_VER || __sun
-#include        <alloca.h>
-#endif
-
-#define null NULL
-typedef unsigned char ubyte;
-typedef unsigned short ushort;
-#define mask(m) (1 << (m))
-
-#if MARS
-#if TARGET_WINDOS
+module dmd.backend.cv8;
+
+version (MARS)
+{
+
+import core.stdc.stdio;
+import core.stdc.stdlib;
+import core.stdc.string;
+extern (C) char* getcwd(char*, size_t);
```

Comment:
`import core.stdc.unistd` ?

Reply:
no such file

Reply:
Right, that is [`core.sys.posix.unistd`](https://github.com/dlang/druntime/search?q=getcwd&unscoped_q=getcwd)

Reply:
Windows isn't Posix, and the whole file is version'd for `Posix`. Dealing with these issues is beyond the scope of this PR, and won't work anyway because DMD is compiled with an older version of DMD that would retain these issues.